<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerspectiveCorrection</class>
 <widget class="QDialog" name="PerspectiveCorrection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>4 intersection points</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_s">
       <property name="text">
        <string>S&amp;tandard</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Minimum line length</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Maximum line gap</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Treshold</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_p">
       <property name="text">
        <string>P&amp;robabilistic</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hough</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="pushButton_el">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_hst">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBox_hlg">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>310</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_hll">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Extend lines</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_h">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QPushButton" name="pushButton_4ip">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Select vertical and 
 horizontal lines</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QPushButton" name="pushButton_vh">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
